ewatercycle/models/abstract.py

```diff
@@ -69,6 +70,17 @@ def set_value(self, name: str, value: np.ndarray) -> None:
         """
         self.bmi.set_value(name, value)
 
+    @abstractmethod
```

Comment:
We could have a default implementation here, but due to different grids of models, best to leave abstract